### PR TITLE
docs: add prom-ocs.md

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

**Comment by robszumski** [comment]
Can we change the filename from `prom-ocs.md` to `prometheus-ocs.md` for max SEO?

File: Documentation/alm/prom-ocs.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Using the Prometheus Open Cloud Service to deploy a Prometheus instance will create the following Kubernetes objects:
+* A Prometheus CRD
+* A Prometheus Service Monitor
+* A Prometheus Stateful Set
+* A Prometheus Secret
+
+## Ingesting metrics
+
+Once enabled, go to *https://<your-cluster-dns>/prometheus* to access Prometheus monitoring for the cluster. Select *Status > Targets* to confirm that Prometheus is correctly configured and ingesting metrics.
```
Comment (robszumski) [inline Documentation/alm/prom-ocs.md]: This is talking about the cluster's prometheus, not the OCS

Comment (zbwright) [reply Documentation/alm/prom-ocs.md]: so, remove the section? or is there another way to test that it's working correctly?

File: Documentation/alm/prometheus-ocs.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Using the Prometheus Open Cloud Service to deploy a Prometheus instance will create the following Kubernetes objects:
+* A Prometheus CRD
+* A Prometheus Service Monitor
+* A Prometheus Stateful Set
+* A Prometheus Secret
+
+## Ingesting metrics
+
+Once enabled, go to *https://<your-cluster-dns>/prometheus* to access Prometheus monitoring for the cluster. Select *Status > Targets* to confirm that Prometheus is correctly configured and ingesting metrics.
```
Comment (brancz) [inline Documentation/alm/prometheus-ocs.md]: Ingress is not automatically setup, as there are too many ways to configure it. This is good, just make sure you mention that one needs to setup some way of accessing the UI first.

File: Documentation/alm/prometheus-ocs.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      <img src="../img/prometheus-targets.png" class="img-responsive">
+    </a>
+  </div>
+</div>
+
+## Using Alertmanager
+
+Use the built-in Tectonic Alertmanager with the Prometheus OCS. Alertmanagers should be shared between Prometheus instances.
+
+For Service discovery, the Prometheus Pod must have permission to access the Kubernetes API in this tectonic-system namespace. Follow the instructions in [Monitoring Applications][monitoring-apps] to create a ClusterRoleBinding to bind the available ClusterRole to an appropriate ServiceAccount.
```
Comment (brancz) [inline Documentation/alm/prometheus-ocs.md]: We actually managed to do this automatically when one creates a Prometheus instance via console, we should point this out. (not sure this is released yet though)

**Pull request merged**